### PR TITLE
Add support for aggregate clusterroles in the helm chart

### PR DESCRIPTION
**Pull request by abyth**
### Type of change

- Enhancement / new feature

### Description

Add support for aggregate clusterroles in the strimzi helm chart (ref: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles) to support strimzi crds in the pre-defined clusterroles `admin`, `edit` and `view`. The clusterroles in question are taken from `packaging/install/strimzi-admin`.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [X] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [X] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

File: packaging/helm-charts/helm3/strimzi-kafka-operator/templates/70-ClusterRole-strimzi-admin.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      - strimzipodsets
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - delete
+      - patch
+      - update
+  {{- end -}}
```
Comment (scholzj) [inline packaging/helm-charts/helm3/strimzi-kafka-operator/templates/70-ClusterRole-strimzi-admin.yaml:53]: Stupid question - should the 2 empty spaces be there? Or should it be aligned with the `if`? (not sure if it actually matters a role in Helm Chart)

**Review by scholzj — APPROVED** [review]
I left one nit. But this looks good and useful. Thanks.

File: packaging/helm-charts/helm3/strimzi-kafka-operator/templates/70-ClusterRole-strimzi-admin.yaml (hunk 1 above)
Comment (abyth) [reply packaging/helm-charts/helm3/strimzi-kafka-operator/templates/70-ClusterRole-strimzi-admin.yaml]: My bad.. pushed another commit that removes the redundant whitespace.

**Review by im-konge — COMMENTED** [review]
Just one comment - should the files be named `070-ClusterRole-strimzi-admin.yaml` instead 70-ClusterRole-strimzi-admin.yaml` to keep it correctly named with other files?

**Comment by scholzj** [comment]
@im-konge Yeah, good catch. I think all the other file names have leading 0.

**Review by im-konge — APPROVED** [review]
LGTM :) Thanks a lot for the PR

**Comment by scholzj** [comment]
Thanks for the PR.

**Pull request merged**